modal-js/src/secret.ts

```diff
@@ -42,4 +43,36 @@ export class Secret {
       throw err;
     }
   }
+
+  /** Create a Secret from a plain object of key-value pairs. */
+  static async fromObject(
```

Comment:
Is `fromObject` a good name for it in JS, @freider ?

Reply:
I sort of think it's ok if the name is different in different languages though? The way I think of it is in Python the secret is created from a `dict` so should be called `from_dict`, and in go from a `Map` so `FromMap`. But I do agree that `object` feels weird!

Reply:
`fromEntries`? `fromKvPairs`?

Reply:
Googling a bit it seems `fromObject` is pretty well established (e.g. https://github.com/protobufjs/protobuf.js/), so I'll go with that for now!